scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -108,7 +108,8 @@ abstract class ScanResultsStorage : PackageBasedScanStorage {
 
         result.onSuccess { results ->
             logger.info {
-                "Read ${results.values.sumOf { it.size }} scan result(s) from ${javaClass.simpleName} in $duration."
+                val count = results.values.sumOf { it.size }
+                "Read $count matching scan result(s) from ${javaClass.simpleName} in $duration."
```

Comment:
While at it, would it make sense to say "Read all..."? Or maybe even "Successfully read all..."

Reply:
What are you trying to say with "all"?

Reply:
That all matching results have been read, not just a subset of the matching results.

Reply:
I think that adds more confusion than clarity, because we actually never deal with "a subset of the matching results". We have "all results" and "matching results" (which is a subset of "all results"), but not "all matching results" and thus also not "a subset of the / all matching results".

Reply:
That's something the reader of the logs doesn't know, because this is only in the code. Anyhow, not worth discussing any further I guess.